docs/journalist.rst

```diff
@@ -1,289 +1,299 @@
 Journalist Guide
 ================
 
-Journalists viewing documents on SecureDrop must connect to the
-respective Source or Journalist Interface using the `Tails operating
-system <https://tails.boum.org/>`__, which your admin should
-have already set up for you.
+This guide presents an overview of the SecureDrop system for a
+journalist. It covers the core functions necessary to start working
+with the platform: logging in securely, viewing documents, editing
+documents, and interacting with sources.
 
-Workflow
---------
+Creating a GPG Key
+------------------
 
-.. todo:: This whole document needs to be reworked, it's all over the
-          place and does not flow from top to bottom.
+Each journalist needs a personal GPG key for encrypting files. A GPG
+key has two parts: a *public key* and a *private key*. The private
+key, used for decryption, stays on the *Journalist Workstation*. The
+public key, used for encryption, is copied to the *Secure Viewing
+Station*.
```

Comment:
I'm not certain we should be listing the personal GPG creation as _required_ at this stage—although we've wanted to add first-class support for per-journalist GPG keys for a while, and #2555 shows our docs need help there. So just mentioning the discrepancy, not blocking merge.